### PR TITLE
az-devops: add page

### PR DESCRIPTION
**Pull request by sannae**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

For: #6141

### CODE REVIEW

File: pages/common/az-devops.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Manage Azure DevOps organizations.
+> Part of `azure-cli`.
+> More information: <https://learn.microsoft.com/en-us/cli/azure/devops?view=azure-cli-latest>.
+
+- Set the Personal Access Token (PAT) to login to a particular organization:
+
+`az devops login --organization {{organization_URL}}`
+
+- Open project in browser:
```
Comment (kbdharun) [inline pages/common/az-devops.md:11]:
```suggestion
- Open a project in the browser:
```

File: pages/common/az-devops.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# az devops
+
+> Manage Azure DevOps organizations.
+> Part of `azure-cli`.
+> More information: <https://learn.microsoft.com/en-us/cli/azure/devops?view=azure-cli-latest>.
+
+- Set the Personal Access Token (PAT) to login to a particular organization:
+
+`az devops login --organization {{organization_URL}}`
```
Comment (kbdharun) [inline pages/common/az-devops.md:9]:
```suggestion
`az devops login --organization {{organization_url}}`
```

File: pages/common/az-devops.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`az devops team list-member --project {{project_name}} --team {{team_name}}`
+
+- Check the Azure DevOps CLI current configuration:
+
+`az devops configure --list`
+
+- Configure the Azure DevOps CLI behavior by setting a default project and a default organization:
+
+`az devops configure --defaults project={{project_name}} organization={{organization_URL}}`
```
Comment (kbdharun) [inline pages/common/az-devops.md:25]:
```suggestion
`az devops configure --defaults project={{project_name}} organization={{organization_url}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**